lib/ecmascript.ts

```diff
@@ -5317,19 +5326,46 @@ function RoundNumberToIncrement(quantity: JSBI, increment: number, mode: Tempora
   let { quotient, remainder } = divmod(quantity, JSBI.BigInt(increment));
```

Comment:
I had some internal tests for RoundNumberToIncrement in the other polyfill in order to more easily test it directly, without going through Temporal entry points - maybe worth porting those over to here?

Reply:
That sounds useful, though I can't find them in the spec repo. Mind linking to them? I'll rebase them in in another PR.

Reply:
Here they are: https://github.com/tc39/proposal-temporal/blob/main/polyfill/test/ecmascript.mjs#L41-L67

Reply:
I have no idea why GitHub search refused to show me that until you linked it to me... thanks :) I'll create a follow-up PR for this.